templates/org/settings/hook_new.tmpl

```diff
@@ -41,7 +45,9 @@
 					{{template "repo/settings/webhook/discord" .}}
 					{{template "repo/settings/webhook/dingtalk" .}}
 					{{template "repo/settings/webhook/telegram" .}}
+					{{template "repo/settings/webhook/teamcity" .}}
 					{{template "repo/settings/webhook/msteams" .}}
+					{{template "repo/settings/webhook/teamcity" .}}
```

Comment:
duplicate?

Reply:
I hate rebasing so much sometimes....resolved!  Thanks for the catch!